index.html

```diff
@@ -51,6 +51,200 @@ <h2>
+                            <li class="can">CANOPY</li>
+                            <li class="back">Back End Dev</li>
+                            <li class="year">2015</li>
+                        </ul>
+                    </div>
+                    <p style="padding-bottom: 78px;">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="languages">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div style="padding-top: 24px; padding-bottom: 20px;">
+                        <button>See project</button>
+                    </div>
+
+               </div>
+                
+            </div>
+
+        </section>
+        <section class="work-section">
+            <div class="page-container">
+                <div class="img.wrap">
+                    <img src="images/Snapshoot Portfolio (2).svg">
+                </div>
+               <div class="details">
+                    <div>
+                        <h2>Tonic</h2>
+                    </div>
+                    <div>
+                        <ul>
+                            <li class="can">CANOPY</li>
+                            <li class="back">Back End Dev</li>
+                            <li class="year">2015</li>
+                        </ul>
+                    </div>
+                    <p style="padding-bottom: 78px;">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="languages">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div style="padding-top: 24px; padding-bottom: 20px;">
+                        <button>See project</button>
+                    </div>
+
+               </div>
+                
+            </div>
+
+        </section>
+        <section class="work-section last-work-section">
+            <div class="page-container">
+                <div class="img.wrap">
+                    <img src="images/Snapshoot Portfolio (3).svg">
+                </div>
+               <div class="details">
+                    <div>
+                        <h2>Multi-Post Stories</h2>
+                    </div>
+                    <div>
+                        <ul>
+                            <li class="can">CANOPY</li>
+                            <li class="back">Back End Dev</li>
+                            <li class="year">2015</li>
+                        </ul>
+                    </div>
+                    <p style="padding-bottom: 78px;">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
- Kindly, use the external style CSS as your precious code reviewer suggested 👌👌 Always use the class attribute for multiple elements (do not use ID selector for multiple elements). 👍 👍